src/søknad/steg/3-barnadine/BarnaDine.tsx

```diff
@@ -31,16 +31,30 @@ const BarnaDine: React.FC = () => {
           <div className="barnekort-wrapper">
             {barna?.map((barn, index) => (
               <Barnekort
-                key={index + barn.fødselsdato}
+                key={index + barn.fødselsdato.verdi}
                 settÅpenModal={settÅpenModal}
                 id={barn.id ? barn.id : ''}
                 navn={barn.navn}
                 fnr={barn.fnr}
                 fødselsdato={barn.fødselsdato}
-                personnummer={barn.personnummer ? barn.personnummer : ''}
+                personnummer={
+                  barn.personnummer && barn.personnummer.verdi
+                    ? barn.personnummer
+                    : {
+                        label: 'Personnummer',
+                        verdi: '',
+                      }
+                }
                 alder={barn.alder}
                 harSammeAdresse={barn.harSammeAdresse}
-                ufødt={barn.ufødt ? barn.ufødt : false}
+                født={
+                  barn.født
+                    ? barn.født
+                    : {
+                        label: 'Er barnet født?',
```

Comment:
Her også! - Bruke intl, tekstid og hentTekst() for å hente label fra tekster.ts ✨

Reply:
Done!